src/app/components/navbar/navbar.component.ts

```diff
@@ -20,6 +21,24 @@ export class NavbarComponent implements OnInit {
   isLoggedIn = false;
   currentUser!: IUser;
 
+  // Navbar for manager
+  isManager(): boolean {
+    return MANAGER.includes(this.currentUser.role);
+  }
+
+  menu(): NbMenuItem[] {
+    if (this.isManager()) {
+      this.items = [
+        { title: 'Profile' },
+        {
+          title: 'My Trips'
+        },
+        { title: 'Logout' }
+      ];
+    }
+    return this.items;
+  }
```

Comment:
this needs to be called when the user changes, because right now it is only updated on refresh:
![bug](https://user-images.githubusercontent.com/26777462/167801678-127e4c50-f117-4fb7-9d26-d3f4042edfd7.gif)

Reply:
Done